addons/FlashCards/src/presenter.js

```diff
@@ -22,6 +22,24 @@ function AddonFlashCards_create(){
         cardsFavourites: null
     };
 
+    let isWCAGOn = false;
+    presenter.speechTexts = {};
+    let KEYBOARD_NAVIGATION_ACTIVE_ITEM_CLASS = "keyboard_navigation_active_element";

```

Comment:
biorąc pod uwagę ile razy korzystasz z nazw klas to proponuję, aby zdefiniować stałą CSS_CLASSES i tą także tam umieścić